airflow/executors/celery_executor.py

```diff
@@ -377,6 +387,39 @@ def _check_for_stalled_adopted_tasks(self):
             for key in timedout_keys:
                 self.change_state(key, State.FAILED)
 
+    @provide_session
+    def _clear_stuck_queued_tasks(self, session=None):
+        """
+        For some reasons, tasks can get stuck in queued state in DB while still not in
+        self.queued_tasks and not in self.running_tasks.
```

Comment:
Could you give here a little explanation, why tasks can get stuck. From the "For some reasons..." it sounds like there is some bug and this here is just a bandage.

Reply:
Updated